libs/designer/src/lib/core/configuretemplate/utils/queries.ts

```diff
@@ -86,7 +87,7 @@ export const getTemplate = async (templateId: string): Promise<ArmResource<any>>
 
 export const useAllConnectors = (operationInfos: Record<string, NodeOperation>) => {
   const allConnectorIds = Object.values(operationInfos).reduce((result: string[], operationInfo) => {
-    const normalizedConnectorId = operationInfo.connectorId?.toLowerCase();
+    const normalizedConnectorId = operationInfo.connectorId?.toLowerCase()?.replace(`_${workflowIdentifier}`, '');
```

Comment:
connectorIds should never have the workflowIdentifier, the bug is somewhere else